lib/server.js

```diff
@@ -865,6 +865,13 @@ WalletService.prototype._selectTxInputs = function(txp, cb) {
     var inputs = sortUtxos(utxos);
 
     var bitcoreTx, bitcoreError;
+    var serializationOpts = {
+      disableIsFullySigned: true,
+    };
+    if (!_.startsWith(txp.version, '1.')) {
+      serializationOpts.disableSmallFees = true;
+      serializationOpts.disableLargeFees = true;
```

Comment:
The problem is in bitcore's implementation. It is using its own estimation of the fee (based on `ceil(size / 1000)`) instead of the one already assigned to the transaction.